### PR TITLE
Working left and right scroll for related items

### PR DESCRIPTION
**Pull request by valpizzo**
Many bug fixes! Should work now.

### CODE REVIEW

File: client/src/RIOC/RightArrow.jsx [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+import { ViewableContext } from './RelatedView';
 
 function RightArrow() {
+  const {
+    viewable, setViewable, position, setPosition,
+    related, end, setEnd,
+  } = useContext(ViewableContext);
+
+  function getNextItem() {
+    console.log('getting next Item at position:', position);
```
Comment (benjaminlthornton) [inline client/src/RIOC/RightArrow.jsx]: Remove unnecessary console logs

**Review by benjaminlthornton — COMMENTED** [review]
Other than a couple of console logs, looks good. I like how the carousel only does an axios get when the user hasn't seen the item before. Once the unnecessary console logs are removed, good to merge.

File: client/src/RIOC/RightArrow.jsx (hunk 1 above)
Comment (valpizzo) [reply client/src/RIOC/RightArrow.jsx]: I have done as requested good sir

**Review by benjaminlthornton — APPROVED** [review]
Console logs removed, cause I said to.

**Pull request merged**